### PR TITLE
add Vim help

### PR DESCRIPTION
**Pull request by Konfekt**

### CODE REVIEW

File: doc/vim-yadi.txt [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    " Try to auto detect and use the indentation of a file when opened. 
+    autocmd BufRead * DetectIndent
+    " Otherwise use file type specific indentation. E.g. tabs for Makefiles
+    " and 4 spaces for Python. This is optional.
+    filetype plugin indent on
+    " Set a fallback here in case detection fails and there is no file type
+    " plugin available. You can also omit this, then Vim defaults to tabs.
+    set expandtab shiftwidth=4 softtabstop=4
+    " You stay in control of your tabstop setting.
+    set tabstop=4
```
Comment (timakro) [inline doc/vim-yadi.txt]: Would be easier to read with empty lines like in the original readme.

File: doc/vim-yadi.txt [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+Take a look at the code (https://github.com/timakro/vim-yadi/blob/main/plugin/yadi.vim#L17-L62),
+it fits on a single page.
+
+--------------------------------------------------------------------------------
+COMMANDS                                                       *vim-yadi-commands*
+
+The `:DetectIndent` command tries to auto detect the indentation style in the
+current buffer.
+If it finds tabs it sets ,
+if it finds n spaces it sets .
```
Comment (timakro) [inline doc/vim-yadi.txt]: Missing text in two lines.

File: doc/vim-yadi.txt [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+If the algorithm can't confidently determine the indentation style no settings
+are changed. It comes naturally to set this as an autocommand as can be seen
+above.
+
+The `:IndentTabs` and  commands explicitly apply the
```
Comment (timakro) [inline doc/vim-yadi.txt]: Missing text.

File: doc/vim-yadi.txt [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+There are a bunch of plugins with DetectIndent-like functionality:
+
+*   DetectIndent (https://github.com/ciaranm/detectindent)
+*   sleuth.vim (https://github.com/tpope/vim-sleuth)
+*   Yaifa (https://github.com/Raimondi/yaifa)
+*   matchindent.vim (https://github.com/conormcd/matchindent.vim)
+*   IndentFinder (https://github.com/ldx/vim-indentfinder)
+
+But none of them let you fall back to file type plugins if detection fails. Most of them force you to set a fallback indentation style. sleuth.vim (https://github.com/tpope/vim-sleuth) is more sophisticated and tries to guess the indentation from surrounding files. It's a different approach which comes at the cost of greater complexity and makes the result rather unpredictable.
```
Comment (timakro) [inline doc/vim-yadi.txt]: This should be wrapped at 78 chars I suppose.

**Review by timakro — CHANGES_REQUESTED** [review]
Thanks. There are some minor issues you'll need to fix before I can merge this.

File: doc/vim-yadi.txt (hunk 4 above)
Comment (Konfekt) [reply doc/vim-yadi.txt]: Thank you for the careful proofreading. All suggestions have been applied

**Pull request merged**